### PR TITLE
Add support setting Storage Account Key via configuration

### LINKED ISSUE #10: Use Azure Storage Account keys instead as well as service principal
Hi!

Would it be possible to add the functionality to use storage account keys as well as service principal (but just one at a time)? I’m specifically interested to use this together with backup/restore of k8s.

It would make deploying much easier since these values can be extracted during deployment with ARM Templates (and service principals can make this a little more complicated).

### PR DESCRIPTION
**Pull request by jaygridley**
This PR resolves #10

### CODE REVIEW

**Comment by gitirabassi** [comment]
@jaygridley Is this ready to be merged?

**Comment by jaygridley** [comment]
> @jaygridley Is this ready to be merged?

Yes, another required approval pending.

**Comment by skriss** [comment]
@carlisia @nrb @ashish-amarnath - gentle reminder to please review this PR!

File: README.md [hunk 1]
```diff
@@ -164,18 +166,41 @@ To integrate Velero with Azure, you must create a Velero-specific [service princ
...
+        AZURE_RESOURCE_GROUP=${AZURE_RESOURCE_GROUP}
+        AZURE_CLOUD_NAME=AzurePublicCloud
+        EOF
+    ```
+
+    > available `AZURE_CLOUD_NAME` values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`, `AzureGermanCloud`
+
+### Set storage account access key
+
+_(Optional) To integrate Velero with Azure using, you can use storage account key instead of service principal._
```
Comment (carlisia) [inline README.md]:
```suggestion
_(Optional) To integrate Velero with Azure, you can use storage account key instead of service principal._
```

**Comment by skriss** [comment]
thanks again @jaygridley!!

**Pull request merged**